src/test/java/net/snowflake/ingest/streaming/internal/FlushServiceTest.java

```diff
@@ -474,7 +478,21 @@ public void testBuildAndUpload() throws Exception {
     channel1.insertRows(rows1, "offset1");
     channel2.insertRows(rows2, "offset2");
 
-    ChannelData<?> channel1Data = testContext.flushChannel(channel1.getName());
+    String name1 = channel1.getName();
+    ChannelData<?> channel1Data;
+    try {
+      channel1Data = testContext.flushChannel(name1);
```

Comment:
This line is very weird.
If I do not add this try/catch block, this line throws NullPointerException but only in PR CI and it is not reproducible locally. Same with the next test. It also fails sporadically in CI for this or next test.
I already tried everything, CI even passes notNull assertion for `testContext` and then throws immediately.
There is also nothing to be null in `flushChannel()` (I thought maybe the stack trace is incorrect).

@sfc-gh-tzhang 
Have you encountered such problem?

Reply:
Ok, now passed after rebase